tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/visibility/Visibility.java

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) 2022 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.ballerinalang.test.visibility;
+
+import org.ballerinalang.test.BCompileUtil;
+import org.ballerinalang.test.CompileResult;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+/**
+ * Test the visibility of non-public symbols inside tests folder.
+ *
+ *  @since 2.0.0
+ */
+public class Visibility {
+
+    @Test(description = "Test ")
```

Comment:
description should explain the test case.

Reply:
Or we can completely remove it

Reply:
Removed